package.json

```diff
@@ -72,7 +72,6 @@
     "video.js": "8.5.2",
     "videojs-contrib-ads": "^6.9.0",
     "videojs-contrib-dash": "^5.1.1",
-    "videojs-contrib-quality-levels": "^4.0.0",
```

Comment:
This was throwing a warning, turns out it is included now in videojs and not needed from our end